dhis-2/dhis-api/src/main/java/org/hisp/dhis/util/DateUtils.java

```diff
@@ -562,6 +562,24 @@ public static boolean dateTimeIsValid(final String dateTimeString) {
     }
   }
 
+  public static boolean hasTimeStamp(final String dateTimeString) {
```

Comment:
Since this is only ever used in tests I would rather write these as an assertion like `assertHasTimestamp` as we do in https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/utils/Assertions.java

Reply:
well, I have just included and so it can be used in the future. It can be added in assertion separately but having it in DateUtils perfectly make sense to me.

Reply:
I thought we agreed on removing this one, right? 😅 as we are not using this in our production code

Reply:
yes.